twisted/conch/ssh/keys.py

```diff
@@ -1118,6 +1205,12 @@ def _toString_AGENTV3(self):
             elif self.type() == 'DSA':
                 values = (data['p'], data['q'], data['g'], data['y'],
                           data['x'])
+            # I assume this is what's needed
+            # But commented out because it hasn't been tested yet.
```

Comment:
I think this comment is out of sync; but generally, don't comment stuff out; http://nedbatchelder.com/text/deleting-code.html :)